changelog.md

```diff
@@ -1,7 +1,99 @@
+# 3.9.0 / 2021-07-09
+
+### New Features
+
+For end users:
+ - Internal links. Contents a user can access can be linked to in comments and notes. An autocompletion popup
+   will help the user create the link as soon as they type `#` and then a word in the title of the content, or
+   its id. The created link will display the title of the document, and will have a specific color showing it is
+   an internal link in notes.
+ - Tags on contents (notes and files). Tagging is done in the new tab between the area that shows the
+   content and the timeline. Tags can be added and removed there. They are per space (spaces have different sets of
+   tags). Tags of aspace can be listed and managed in the space settings accessed from the dashboard.
+   Readers cannot add or remove tags to contents or to space. Contributor can add/remove tags to/from contents, but
+   cannot manage tags of a space. Space and content managers can manage tags of a space. Contents can be searched
+   by tags when the advanced search feature is enabled.
+ - Revamped app appearance (size and headers). Formerly, apps showing notes, topics and files didn't take the whole
+   space vertically and horizontally. They also had both a header and a toolbar. The header had a distinctive color.
+   The space and screen estate is now better used. Apps now take as much space as possible, both vertically and
+   horizontally, to put the focus on the content. The header and the toolbar have been merged and the specific color
+   in the header is now only used for the icon.
```

Comment:
```suggestion
   horizontally, to put the focus on the content.
```
Just a suggestion, you can resolve if you don't want to apply: I wouldn't put that phrase. Color is also present on buttons, on the timeline...

Reply:
it is present at those places indeed, but the sentence is about the header only. I hesitated too.